consent-ws/src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -371,8 +376,8 @@ public Response cancelDataAccessRequest(@PathParam("referenceId") String referen
     }
 
     @GET
-    @Produces("application/json")
-    @Path("/hasUseRestriction/{referenceId}")
+    @Consumes("application/json")
+    @Produces("application/json")@Path("/hasUseRestriction/{referenceId}")
```

Comment:
nit: split into two lines
```suggestion
    @Produces("application/json")
    @Path("/hasUseRestriction/{referenceId}")
```